gcs_to_bq/common/census_to_bq.py

```diff
@@ -0,0 +1,72 @@
+import csv
+import os
+import json
+import time
+from google.cloud import bigquery, storage
+from google.cloud.storage import Blob
+from .json_tools import JsonTable, to_dict
+
+def write_state_names_to_bq(table_name, gcs_bucket, filename):
+  """Writes state names to big query from bucket
+     
+     table_name: The name of the biquery table to write to
+     gcs_bucket: The name of the gcs bucket to read the data from
+     filename: The name of the file in the gcs bucket to read from"""
+  client = bigquery.Client()
+  # TODO replace this with the dataset we'll end up actually using
+  dataset_ref = client.dataset('aaron_census_test')
+  job_config = bigquery.LoadJobConfig()
+  
+  # Always append, so we can keep the history.
+  job_config.write_disposition = 'WRITE_APPEND'
```

Comment:
You can use the member bigquery.WriteDisposition.WRITE_APPEND instead

Reply:
Done!